packages/EasySchedule/src/ScheduleRunner.php

```diff
@@ -50,6 +45,7 @@ public function run(ScheduleInterface $schedule, OutputInterface $output): void
                 $event->run($schedule->getApplication());
             } finally {
                 $lock->release();
+                $this->eventDispatcher->dispatch(new CommandExecutedEvent($event));
```

Comment:
I think we should dispatch the events before releasing the lock to prevent side effects in applications in case of multiple instances running scheduled commands.

Reply:
Done